orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/ExplicitRollback.groovy

```diff
@@ -54,9 +55,24 @@ class ExplicitRollback implements Rollback {
   ApplySourceServerGroupCapacityStage applySourceServerGroupCapacityStage
 
   @JsonIgnore
-  def List<Stage> buildStages(Stage parentStage) {
-    def stages = []
+  List<Stage> buildStages(Stage parentStage) {
+    Map disableServerGroupContext = new HashMap(parentStage.context)
+    disableServerGroupContext.serverGroupName = rollbackServerGroupName
+    def disableServerGroupStage = newStage(
+      parentStage.execution, disableServerGroupStage.type, "disable", disableServerGroupContext, parentStage, SyntheticStageOwner.STAGE_AFTER
+    )
 
+    def parentDeployStage = getParentDeployStage(parentStage)
+    def parentDeployStageStrategy = parentDeployStage?.context?.strategy as String
+    if (Strategy.fromStrategy(parentDeployStageStrategy) == Strategy.ROLLING_RED_BLACK) {
+      // no need to do anything but disable the newly deployed (and failing!) server group when dealing with a
```

Comment:
this _only_ kicks in when we're automatically rolling back something that was created as part of a rolling red/black ... they'll be a check box on the deploy configuration that enables automatic rollback